### DISCUSSION TITLE
Future ready talent

### DISCUSSION THREAD

**Discussion by divyavemasani2910** [github/india#310] [Students]
I have completed all my industry sections but still its showing red...may i know the reason y?

**Comment by Phoenixcoder-6** [comment]
Hey there!! I have also enrolled for the internship program and I am facing the same issue, I think the problem is from their side, even the self learning path also has problems.

**Reply by divyavemasani2910** [reply]
yeah i have mailed to support team regarding this but there is no response from them

**Reply by divyavemasani2910** [reply]
Now problem rectified once check your portal

**Reply by aasthadwivedi** [reply]
> yeah i have mailed to support team regarding this but there is no response from them

I am facing the same problem. I can't even access the Learning page, could you help me please?

**Reply by Phoenixcoder-6** [reply]
You can use Incognito mode or Microsoft Edge.. I used Microsoft Edge for Industrial sessions.. it worked.

**Reply by divyavemasani2910** [reply]
Make sure that you should watch the video in one go and login in microsoft edge

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by Soumajit-Roy** [comment]
> 
> 
> I have completed all my industry sections but still its showing red...may i know the reason y?

Did You watch the videos without skipping,forwarding,muting or fullscreenig in a one go?

**Reply by divyavemasani2910** [reply]
> > I have completed all my industry sections but still its showing red...may i know the reason y?
> 
> Did You watch the videos without skipping,forwarding,muting or fullscreenig in a one go?

yes

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by Kittu-2208** [comment]
I'm facing the same issue

**Reply by divyavemasani2910** [reply]
watch industry sessions in one go and open it in microsoft edge...self learning session update progress in one week those will take time

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
